parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -495,6 +495,10 @@ write_files:
         [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.untrusted.options]
           BinaryName = "/usr/bin/runc"
         {{- end}}
+        {{- if IsKata }}
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
```

Comment:
name of runtime (class)? 

custom binary path?

Reply:
I don't know how to test this without binaries, I guess we'll have to fix it when we get the vhd